### PR TITLE
Fix serial console

### PR DESCRIPTION
**Pull request by telphan**

### CODE REVIEW

File: layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/beaglebone-integrate-with-resin-configuration.patch [hunk 1]
```diff
@@ -32,19 +32,19 @@ index e1e89c4..b7af3ad 100644
...
+@@ -10,9 +10,9 @@
  	"mmcdev=0\0" \
  	"mmcrootfstype=ext4 rootwait\0" \
  	"finduuid=part uuid mmc ${bootpart} uuid\0" \
 -	"args_mmc=run finduuid;setenv bootargs console=${console} " \
 +	"args_mmc=setenv bootargs console=${console} " \
  		"${optargs} " \
 -		"root=PARTUUID=${uuid} rw " \
-+		"${resin_kernel_root} " \
++   "${resin_kernel_root} " \
```
Comment (floion) [inline layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/beaglebone-integrate-with-resin-configuration.patch]: It looks like this change breaks indentation

**Comment by agherzan** [comment]
These changes touch the uboot so we need to make sure resinhup interface remains valid as per @michal-mazurek 's test. https://github.com/resin-os/resinos-tests/blob/master/tests/all/resinhup/fingerprints . As soon as this happens can be merged.

**Comment by agherzan** [comment]
The link above is wrong. This is the correct one : https://github.com/resin-os/resinos-tests/tree/master/tests/all/resin_root_part. Sorry about that.

**Comment by telphan** [comment]
Test works fine.

**Comment by agherzan** [comment]
@telphan any idea why build is not successfully finished?

**Comment by telphan** [comment]
Jenkins was broken

**Comment by agherzan** [comment]
Do retry then.

**Pull request merged**